2019 08 14 changes

jacklinw opened the pull request.

jacklinw committed:
Add audio fades to macro 1 and 10 for between services

Camera 17 is the announcement rotation, which is embedded with Stream BGM. We fade this up and the master down between services so we do not hear the audience having full conversations.

We fade up on macro 1 for the countdown, instead of at macro 2, because often the band will come in early before the countdown is over, and we need to be able to hear them.

jacklinw committed:
Set macro 1 to preview camera 9 as the last step

Often, directors forget to preview something useful when they hit macro 2 to start the service. Preview with a decent and safe camera so it's not a disaster when it happens

tfe commented on Sunday Default.xml:
```diff
@@ -328,15 +428,105 @@
+            -->
+            <Op id="MacroSleep" frames="2"/>
+            <Op id="AudioMixerInputGain" input="Camera17" gain="2429"/>
+            <Op id="AudioMixerMasterOutGain" gain="7682"/>
```
Shouldn't this be the XLR input?

jacklinw committed:
Correct the master fader and remove crossfade in annc rot

The master record mix is the ExternalXLR input, not the master output.

AFV will turn on the audio for a video source when we switch to that camera. Tested with AnncRotation, we do not need to cross-fade the audio for that channel down; instead we are leaving it at the default audio level

tfe commented on Sunday Default.xml:
```diff
@@ -73,11 +93,56 @@
             <Op id="HyperDeckRecord" hyperDeckIndex="2"/>
-            <Op id="MacroSleep" frames="15"/>
+            <!-- Frames: 15
+                - Fade Master to 0
```
I would call this “unity” to disambiguate from it being off.

tfe commented on Sunday Default.xml:
```diff
@@ -90,6 +155,11 @@
             <Op id="MediaPlayerLoop" mediaPlayer="3" loop="True"/>
             <Op id="MediaPlayerPlay" mediaPlayer="3"/>
+            <!-- Audio Setting
+                - Fade AnncRot/StreamBGM to neg-inf
```
This comment is left over now right?

tfe reviewed (approved):
Looks good aside from some things in the comments.

jacklinw committed:
Rename 0 to 'unity' and remove an annc rotation comment
